styles.css

```diff
@@ -150,10 +152,455 @@ nav {
+  top: 385px;
+  padding: 0 24px 0 24px;
+  order: 1;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.card-works {
+  height: 386px;
+  width: 327px;
+  left: 24px;
+  background-image: url(img/Img-Placeholder1.png);
+  display: flex;
+  flex-direction: column;
+  gap: 26px;
+  padding-top: 22%;
+  text-align: inherit;
+  text-decoration: #fff;
+  margin-left: 1px;
+}
+
+.card-title {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 34px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #fff;
+  padding-left: 3%;
+  padding-right: 3%;
+  position: relative;
+  gap: 13%;
+  display: flex;
+}
+
+.card-description {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-left: 3%;
+  padding-right: 3%;
+  color: #fff;
+}
+
+.card-btn {
+  height: 48px;
+  width: 327px;
+  background: #ff6b00;
+  position: relative;
+  top: 3.12%;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #fff;
+  border: 0;
+}
+
+.card-techskills {
+  display: flex;
+  flex-direction: row;
+  gap: 10%;
+  height: 40px;
+  padding-left: 3%;
+  width: 70%;
+  align-items: center;
+}
+
+.card-skillset {
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+  box-sizing: border-box;
+  padding: 10px;
+}
+
+.about {
+  position: absolute;
+  width: 100%;
+  height: 1801px;
+  top: 528%;
+  background-color: #1c1a19;
+  background-image: url(img/Frame.png);
+  background-attachment: scroll;
+  background-size: 41.9%;
+  background-position-y: 24%;
+  background-repeat: no-repeat;
+  background-position-x: 100%;
+}
+
+.about-me {
+  position: inherit;
+  top: 120px;
+  left: 24px;
+  display: flex;
+  flex-direction: column;
+  gap: 24%;
+}
+
+.about-title {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 72px;
+  font-weight: 400;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.about-text {
+  height: 148px;
+  width: 327px;
+  color: #f4f5f7;
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.resume {
+  height: 48px;
+  width: 156px;
+  left: 24px;
+  top: 395px;
+  border-radius: 0;
+  padding: 12px;
+  color: #fff;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  border: none;
+}
+
+.divider {
+  color: #fff;
+  position: relative;
+  top: 36%;
+  margin-left: 12%;
+  margin-right: 12%;
+  border: 1px solid #fff;
+}
+
+.profession-tools {
+  height: 1080px;
+  width: 100%;
+  position: absolute;
+  top: 4580px;
+  padding: 120px 24px 38px 24px;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+}
```

Comment:
- [ ] As suggested by the previous reviewer please make sure you are following the layout of the[ figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=48-27&t=V8Y95Xw5Xt48Kl9N-0)and there is an appropriate margin between the technology categories ("Languages", "Frameworks" and "Skills") and the about section `<hr>` 

|figma design| Your design|
|-------------|-------------|
|<img width="488" alt="Screenshot 2023-05-06 at 2 15 53 PM" src="https://user-images.githubusercontent.com/44798044/236615583-de5b0f2c-c5bb-4a12-8feb-d2d406557814.png">|<img width="400" alt="Screenshot 2023-05-06 at 1 59 46 PM" src="https://user-images.githubusercontent.com/44798044/236615685-d1ff93c6-12bd-4069-b440-c44172557c28.png">|